docs/DeveloperGuide.md

```diff
@@ -376,24 +377,96 @@ list to be displayed correctly on the user's screen.
         * Users might be able to execute commands on entries in the list that are not currently visible, which might lead
           to confusion.
 
+### Internal implementations of Buyers and Properties
+
+Cobb allows functionality that performs operations on two distinct types of entries stored in the database: Potential property
+buyers, as well as properties that are for sale. To allow for this functionality, two classes were created that represented each type
+of entry: `Buyer` and `Property`, respectively.
+
+The structure of a `Buyer` object can be viewed in the class diagram below.
+
+![BuyerClassDiagram](images/BuyerClassDiagram.png)
+
+From the diagram, it can be seen that a `Buyer` object consists of the following attributes:
+- A `Name`, representing the name of the buyer.
+- A `Phone`, representing the phone number of the buyer.
+- An `Address`, representing the address of the buyer.
+- An `Email`, representing the email address of the buyer.
+- An `entryTime`, representing the time of creation of the buyer (created and accessed internally).
```

Comment:
Should we change the naming convention for all entry times? Since I changed it based on your suggestion for the error messages and UG